Application/EdFi.Ods.AdminApp.Management.Tests/ClaimSetEditor/DeleteClaimSetCommandV6ServiceTests.cs

```diff
@@ -99,92 +96,92 @@ public void ShouldThrowExceptionOnEditSystemReservedClaimSet()
-            validationResults.Errors.Single().ErrorMessage.ShouldBe("Only user created claim sets can be deleted");
-        }
-
-        [Test]
-        public void ShouldNotDeleteClaimSetIfNotAnExistingId()
-        {
-            var testApplication = new Application
-            {
-                ApplicationName = "TestApplication2"
-            };
-            Save(testApplication);
-
-            var testClaimSet = new ClaimSet { ClaimSetName = $"TestClaimSet{DateTime.Now:O}", Application = testApplication };
-            Save(testClaimSet);
-
-            var claimSetToDelete = new DeleteClaimSetModel()
-            {
-                Name = testClaimSet.ClaimSetName,
-                Id = 99,
-                IsEditable = true
-            };
-            using var securityContext = TestContext;
-            var getClaimSetByIdQuery = ClaimSetByIdQuery(securityContext);
-            var validator = new DeleteClaimSetModelValidator(getClaimSetByIdQuery);
-            var validationResults = validator.Validate(claimSetToDelete);
-            validationResults.IsValid.ShouldBe(false);
-            validationResults.Errors.Single().ErrorMessage.ShouldBe("No such claim set exists in the database");
-        }
-
-        [Test]
-        public void ShouldNotDeleteClaimSetHasAnAssociatedApplication()
-        {
-            var testApplication = new Application
-            {
-                ApplicationName = "TestApplication3"
-            };
-            Save(testApplication);
-
-            var testClaimSet = new ClaimSet { ClaimSetName = $"TestClaimSet{DateTime.Now:O}", Application = testApplication };
-            Save(testClaimSet);
-
-            var claimSetToDelete = new DeleteClaimSetModel()
-            {
-                Name = testClaimSet.ClaimSetName,
-                Id = testClaimSet.ClaimSetId,
-                IsEditable = true,
-                VendorApplicationCount = 1
-            };
-
-            using var securityContext = TestContext;
-            var getClaimSetByIdQuery = ClaimSetByIdQuery(securityContext);
-            var validator = new DeleteClaimSetModelValidator(getClaimSetByIdQuery);
-            var validationResults = validator.Validate(claimSetToDelete);
-            validationResults.IsValid.ShouldBe(false);
-            validationResults.Errors.Single().ErrorMessage.ShouldBe($"Cannot delete this claim set. This claim set has {claimSetToDelete.VendorApplicationCount} associated application(s).");
+            exception.Message.ShouldBe($"Claim set `{systemReservedClaimSet.ClaimSetName}` cannot be deleted.");
         }
 
-        private GetClaimSetByIdQuery ClaimSetByIdQuery(ISecurityContext securityContext) => new GetClaimSetByIdQuery(new StubOdsSecurityModelVersionResolver.V6(),
-                        null, new GetClaimSetByIdQueryV6Service(securityContext));
+        // TODO: move these to UnitTests, using appropriate validator from the API project
```

Comment:
I'll handle this TODO and a few others in one of the next tasks I work on in this epic.